pkg/query/query.go

```diff
@@ -312,6 +312,14 @@ func (q *Query) renderReport(ctx context.Context, p profile.InstantProfile, typ
 		return &pb.QueryResponse{
 			Report: &pb.QueryResponse_Pprof{Pprof: buf.Bytes()},
 		}, nil
+	case pb.QueryRequest_REPORT_TYPE_TOP:
+		top, err := GenerateTopTable(ctx, q.metaStore, p)
```

Comment:
Where is `GenerateTopTable` defined? I can't find it? :thinking:

Reply:
Seems like I missed to commit the untracked files :see_no_evil: